### PR TITLE
boards/nucleo-common: add saul support for on-board user led and button

### PR DESCRIPTION
**Pull request by aabadie**
Did that in #5706 but as this change is quite independent and fairly straight forward, it's better to open a dedicated PR for it.

### CODE REVIEW

**Comment by aabadie** [comment]
Tested on several nucleo boards: nucleo-f072, nucleo-f401, nucleo-f091. And also on nucleo-f446 (in #5706) and nucleo-f411 (PR should come soon).

**Comment by PeterKietzmann** [comment]
@aabadie I don't have an overview about all these board. This is a general question: is it reasonable to define the `LED0_PIN` in the common folder (and thus the saul support)? I'm wondering if the LED is always connected to `PA5` and also if it is always green.

File: boards/nucleo-common/include/gpio_params.h [hunk 1]
```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (C) Inria 2016
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup   boards_nucleo-f446
```
Comment (PeterKietzmann) [inline boards/nucleo-common/include/gpio_params.h]: I assume it should rather be group `boards_nucleo_common` or so?

Comment (aabadie) [inline boards/nucleo-common/include/gpio_params.h]: Yes, I just forgot to update this change.

**Comment by aabadie** [comment]
>  I'm wondering if the LED is always connected to PA5 and also if it is always green.

For the moment, I just verified this PR with already supported nucleo boards. The LED was always green and could be controlled with the same code. One should look in the datasheet to be sure it can be generalized to all other nucleo boards.

**Comment by PeterKietzmann** [comment]
Fine. Would you please squash the second commit?

**Comment by aabadie** [comment]
@PeterKietzmann, commit squashed.

**Comment by aabadie** [comment]
@PeterKietzmann, Murdock is happy now.

**Comment by PeterKietzmann** [comment]
ACK and go

**Pull request merged**